src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -132,7 +132,6 @@ class boss_nazan : public CreatureScript
 
                 if (flight) // phase 1 - the flight
                 {
```

Comment:
```suggestion
                {
                    Creature* Vazruden = ObjectAccessor::GetCreature(*me, VazrudenGUID);
```
Not sure which one you like better, mark the one you don't want as Resolved (if you have got access to that feature button).